src/components/MyReservations.js

```diff
@@ -1,9 +1,56 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import axios from 'axios';
+import { useSelector, useDispatch } from 'react-redux';
+import Reservation from './Reservation';
+import { setData } from '../redux/Reservations/reservation';
 
-const MyReservations = () => (
-  <section>
-    <h1>My Reservation</h1>
-  </section>
-);
+const MyReservations = () => {
+  const reservations = useSelector((state) => state.allReservation.reservation);
+  const dispatch = useDispatch();
+  const fetchReservations = async () => {
+    const response = await axios.get('http://[::1]:4000/api/v1/reservation');
+    dispatch(setData(response.data.data));
+  };
+  useEffect(() => {
+    fetchReservations();
+  }, []);
+
+  return (
+    <section>
+      <table className="table table-borderless w-75">
+        <thead>
+          <tr>
+            <th>Id</th>
+            <th>City</th>
+            <th>Data</th>
+            <th>Car</th>
+            <th className="d-flex justify-content-center">Operation</th>
+          </tr>
+        </thead>
+        <tbody>
+          {
+            reservations.map((reservation) => {
+              /* eslint-disable */ 
+
+              const {
+                car_id, id, city, date,
+              } = reservation;
+              return (
+                <Reservation
+                  key={reservation.id}
+                  id={id}
+                  city={city}
+                  date={date}
+                  carId={car_id}
+                />
+              );
+            })
+}
```

Comment:
![no-reservation-created-in-redux (copy)](https://user-images.githubusercontent.com/93039359/176023187-94e9baaa-fc96-4ad1-a4b8-093630f0d306.png)

- [ ] I have created a reservation. It made a call to the API but didn't store it in the redux 
store. Kindly fix this issue.